docs/source/about/cite.rst

```diff
@@ -0,0 +1,37 @@
+How to cite us
+==============
+If you find CellRank useful for your research, please consider citing :cite:`lange:22` as:
+
+.. code-block:: bibtex
+
+    @article{lange:22,
+        title = {CellRank for directed single-cell fate mapping},
+        author = {Lange, Marius and Bergen, Volker and Klein, Michal and Setty, Manu and
+                  Reuter, Bernhard and Bakhti, Mostafa and Lickert, Heiko and
+                  Ansari, Meshal and Schniering, Janine and Schiller, Herbert B. and
+                  Pe'er, Dana and Theis, Fabian J.},
+        journal = {Nat. Methods},
+        year = {2022},
+        doi = {10.1038/s41592-021-01346-6},
+        publisher = {Nature Publishing Group}
+    }
+
+In addition, if you use the :class:`cellrank.estimators.GPCCA` to compute initial, terminal or intermediate states,
+please consider citing :cite:`reuter:18` as:
```

Comment:
Hi @msmdev , I'll go over this in more detail later; for now, I just wanted to mention that we don't use GPCCA to compute fate probabilities. That's done purely on the CellRank side and does not rely on any GPCCA theory. Users can manually set terminal states and compute fate probabilities towards them without ever using pyGPCCA; therefore, I wil not mention fate probabilities here.

Reply:
Also, I'm not a fan of the formulation "you are actually using XXX" - they *are* using CellRank, and in the backend, the computation is transferred to pyGPCCA. Also, I think it's counter-productive to make users cite two works for GPCCA, given that one of them is a software. We also don't require them to cite the CellRank software package separately.

Reply:
Re `cellrank.tl` function; as discussed in person and via email, we are removing these high level functions in favour of the low-level interaction with esimators.

Reply:
I'm happy to cite the 2019 paper here rather than the 2018 paper.

Reply:
@Marius1311: Ok, so cross out the fate probabilities but you could keep the absorption probabilities (since they are an attribute of the GPCCA estimator). Actually, I assumed fate probs to be synonymous to absorption probs, but if it isn't the case I had a misconception here.

Regarding your problem with "you are actually using pyGPCCA": Feel free to suggest an alternative formulation. I just wanted to say that they are using pyGPCCA under the hood here (wapped by CellRank). One should - to be fair - clearly emphasize here that the actual computation is done by pyGPCCA.

Regarding citing pyGPCCA: I just added this as an option as is stated in "Please consider citing the pyGPCCA package :cite:reuter:22 as well". This simply originates from the fact that there is no paper introducing the pyGPCCA software - in contrary to the CellRank paper. Maybe we could formulate it more clearly as a "nice to have" option, but if you really think it is counter-productive we might it.

Regarding `cellrank.tl ` removal: Does this mean that you remove `cellrank.initial_states` and `cellrank.terminal_states` completely in favor of adding them to `cellrank.estimators.GPCCA` as attributes?

Reply:
My adapted suggestions:
"If you use :class:cellrank.estimators.GPCCA to compute initial, terminal or intermediate states or absorption probabilities, you are using the [pyGPCCA](https://github.com/msmdev/pyGPCCA) package, which implements the Generalized Perron Cluster Cluster Analysis (GPCCA), in the background. Thus, you should cite GPCCA :cite:reuter:19. You might consider citing the pyGPCCA package :cite:reuter:22 as well."
or
"If you use :class:cellrank.estimators.GPCCA to compute initial, terminal or intermediate states or absorption probabilities, you are using the [pyGPCCA](https://github.com/msmdev/pyGPCCA) package :cite:reuter:22, which implements the Generalized Perron Cluster Cluster Analysis (GPCCA), in the background. Thus, you should cite GPCCA :cite:reuter:19."

Reply:
Hi @msmdev, I adapted the phrasing according to your suggestions. Re your questions:
- removing `cr.tl.anything`: yes, will be removed in 2.0 in favor of estimator-level access. We won't add them as novel attributes to the GPCCA estimator - all the functionality is already there (and has always been). However, we will rename some current methods/attributes to make usage somewhat more intuitive. 
- Absorption vs. fate probabilities: they are the same thing, I use the two terms interchangeably. However, neither of them uses any pyGPCCA code or theory, feel free to check in code. They are implemented in the `AbsProbsMixin`class and don't rely on pyGPCCA - they are pure CellRank methods. We just compute absorption probs of a Markov chain, we developed this separately to our collab about GPCCA and it does not have anything to do with pyGPCCA.  Accordingly, if someone just uses the GPCCA estimator to compute absorption probabilities towards predefined terminal states (allowed in CellRank), they are not using the GPCCA method or the pyGPCCA package.

Reply:
@Marius1311, thank you very much for the kind adaptation!

Regarding absorption/fate probabilities: Sure, there are several ways to compute absorption probabilities to *predefined* states. But, if you are using GPCCA to cluster the microstates into macrostates and define initial, intermediate and terminal states and then use those terminal states to compute absorption probabilities (with whatever method), you are actually using the information gained via GPCCA (the terminal states) to calculate your coarse grained absorption probabilities. So, if you are using the cellrank.estimators.GPCCA.absorption_probabilities attribute, you are using GPCCA implicitly. Or do I get this wrong? @Marius1311, could you please elaborate? Thanks!